$OpenBSD: patch-_metadata,v 1.2 2018/02/23 11:23:45 sebastia Exp $

Switch mini-portile2 to a development depedency, it is only needed for
building, and not even needed there when using --use-system-libraries.

Index: .metadata
--- .metadata.orig
+++ .metadata
@@ -18,7 +18,7 @@ dependencies:
     - - "~>"
       - !ruby/object:Gem::Version
         version: '2.3'
-  type: :runtime
+  type: :development
   prerelease: false
   version_requirements: !ruby/object:Gem::Requirement
     requirements:
